app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -382,23 +382,23 @@ class Settings(private val appContext: Context) : PreferencesHolder {
      * four hours of inactivity.
      */
     var startOnHomeAfterFourHours by booleanPreference(
-        appContext.getPreferenceKey(R.string.pref_key_start_on_home_after_four_hours),
+        appContext.getPreferenceKey(R.string.pref_key_opening_screen_homepage),
         default = true
     )
 
     /**
      * Indicates if the user has selected the option to always start on the home screen.
      */
     var startOnHomeAlways by booleanPreference(
-        appContext.getPreferenceKey(R.string.pref_key_start_on_home_always),
+        appContext.getPreferenceKey(R.string.pref_key_opening_screen_last_tab),
         default = false
     )
 
     /**
      * Indicates if the user has selected the option to never start on the home screen.
      */
     var startOnHomeNever by booleanPreference(
-        appContext.getPreferenceKey(R.string.pref_key_start_on_home_never),
+        appContext.getPreferenceKey(R.string.pref_key_opening_screen_after_four_hours_of_inactivity),
         default = false
```

Comment:
As we are changing they keys, should we also change the variable names and the docs to reflect the new behavior? :)